scripts/scene/RoomBuilder.gd

```diff
@@ -3,6 +3,7 @@ class_name RoomBuilder extends Node
 const ROOM_SCENE: PackedScene = preload("res://room.tscn")
 
 signal action_completed(action: int)
+signal room_built(room_type, tiles)
```

Comment:
@conalli @jazmand This is where I am struggling. I do not know the correct place to define the signal and how to set it.

I want the confirmation of room building to trigger the start of furniture placement. I thought the problem might be that I needed to define FurnitureManager.new() in the GameManager, but I cannot see use doing that for BuildingManager.

Could we take a look together?